test-app/led.c

```diff
@@ -146,6 +146,43 @@ void boot_led_on(void)
 
 #endif /** PLATFORM_stm32g0 **/
 
+#ifdef PLATFORM_stm32c0
```

Comment:
At quick glance is identical to the above `PLATFORM_stm32g0` section. I am okay with reusing the same code for both. `#if defined(PLATFORM_stm32g0) || defined(PLATFORM_stm32c0)`. Looks like it ended up being GPIOA5 for both.